frontend/src/pages/WaitingPage.tsx

```diff
@@ -45,14 +47,19 @@ export const WaitingPage = () => {
       setTimeout(() => navigate("/dashboard"), 3000);
     }, 30000);
 
-    socket.on("found", (data) => {
-      setMessage("Found a match! Loading...");
+    socket.on("found", (match: Match) => {
+      setMessage(
+        `Found a match between ${match.result[0].userId} and ${match.result[1].userId}! \n
+        Room ID: ${match.roomId} \n
+        Loading...`,
+      );
```

Comment:
The backtick notation doesn't require newlines.
```suggestion
      setMessage(
        `Found a match between ${match.result[0].userId} and ${match.result[1].userId}!
        Room ID: ${match.roomId}
        Loading...`,
      );
```